Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -979,6 +996,7 @@ void TextureCache::CopyRenderTargetToTexture(u32 dstAddr, unsigned int dstFormat
 	unsigned int scaled_tex_w = g_ActiveConfig.bCopyEFBScaled ? Renderer::EFBToScaledX(tex_w) : tex_w;
 	unsigned int scaled_tex_h = g_ActiveConfig.bCopyEFBScaled ? Renderer::EFBToScaledY(tex_h) : tex_h;
 
+	// TODO: do we need to free these textures at the start, or is make range dynamic enough?
```

Comment:
Well, the code paths are merged in #2960, which is why that TODO got added.

And yes, it breaks NSMBW partial updates.